plugins/webp-uploads/settings.php

```diff
@@ -113,13 +118,29 @@ function webp_uploads_generate_avif_webp_setting_callback(): void {
 
 	$selected       = webp_uploads_get_image_output_format();
 	$avif_supported = webp_uploads_mime_type_supported( 'image/avif' );
-	// Ensure WebP selected if AVIF is not supported.
-	if ( ! $avif_supported ) {
+	$webp_supported = webp_uploads_mime_type_supported( 'image/webp' );
+
+	// If neither format is support, the entire field is not shown.
+	if ( ! $avif_supported && ! $webp_supported ) {
+		?>
+		<br />
+		<div class="notice notice-warning is-dismissible inline">
```

Comment:
Probably shouldn't be dismissible?

```suggestion
		<div class="notice notice-warning inline">
```

Reply:
right